src/vmm/src/guest_config/aarch/mod.rs

```diff
@@ -0,0 +1,18 @@
+// Copyright 2023 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
Is there a reason to drop `64` from the names of the modules?

Reply:
+1

Reply:
Just that the notion of supporting 32-bit is not relevant in the Firecracker context.

Reply:
The problem is there is no such thing as just `aarch` (like `arm`, for example). It must be either `aarch64` or `aarch32`, but not just `aarch`, otherwise it becomes rather confusing.

Reply:
Also we already have aarch64 naming in use